### ISSUE TITLE
run.sh: line 6: stdbuf: command not found

### ISSUE THREAD

**Issue by connormcmk** [balena-io-playground/ssh-key-insert#6]
uname -a: 
Darwin My-MacBook-Pro 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64

./ssh-key-insert » ./run.sh          
./run.sh: line 6: stdbuf: command not found

Seems like stdbuf is not a native OSX command. I have gnu coreutils installed, but that puts it under `gstdbuf`

`which gstdbuf`
/usr/local/bin/gstdbuf

Can this be made to be OSX compatible? 

It seems that there is also a tee incompatibility:
tee: illegal option -- -
usage: tee [-ai] [file ...]

For now, I solved this with:
`gsed -i 's+stdbuf+gstdbuf+g' run.sh && gsed -i 's+tee+gtee+g' run.sh`

Then `./run.sh` worked. (if you don't have gnu coreutils install it with `brew install coreutils`)

**Comment by imrehg** [comment]
Hey @cnrmck, yeah, this is more tested on Linux, and don't think it would at all work on Windows for now. Good info about OSX.

We'll see how we can add your changes in a way, that it works on both platforms. Since it's our "playground" collection of projects, they are in flux, and and might not be fit for all the different ways people can use them. E.g. the problem that this script is trying to solve should in general be part of the platform for example (and that's the plan) so that the tools is not needed anymore.

**Comment by connormcmk** [comment]
Yeah that makes sense. I’m glad to hear that the plan is to integrate these
features into the core product.

Seems like a smart strategy: make a repo to solve a problem, as issues
arise, change that small repo (not a whole production app), if it gets very
popular then take that as a signal it should be integrated into the main
product.

Brilliant

Just wanted to share my solution for anyone using Mac. If it would help I
can make a quick pull request that will solve the problem.
